cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -28,7 +28,13 @@
   {% endif -%}
 {% endif -%}
 
-{% set params = "--master=127.0.0.1:8080" + " " + cluster_name + " " + cluster_cidr + " " + allocate_node_cidrs + " " + cloud_provider  + " " + cloud_config + service_account_key + pillar['log_level'] -%}
+{% set root_ca_file = "" -%}
+
+{% if grains['cloud'] is defined and grains.cloud in [ 'aws', 'gce' ]    %}
```

Comment:
@errordeveloper @squillace @chanezon @crossorigin would you guys like me to add 'azure' as a cloud provider that supports this? We only need a valid Root CA for apiserver in /srv/kubernetes/ca.crt of the root fs where controller-manager is running. I can enable this or if this is not ready, you can add it later.